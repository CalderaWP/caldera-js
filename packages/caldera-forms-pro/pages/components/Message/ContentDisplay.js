import React from 'react';

export const ContentDisplay = ({content}) => {
	const createMarkup = () => {
		const {rendered} = content;
		return {__html:rendered};
	};
	return <div dangerouslySetInnerHTML={createMarkup()}/>


};

