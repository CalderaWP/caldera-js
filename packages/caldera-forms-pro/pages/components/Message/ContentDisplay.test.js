import {ContentDisplay} from "./ContentDisplay";
import React, {Fragment} from 'react';
import renderer from 'react-test-renderer';
import {mount} from 'enzyme';
describe( 'ContentDisplay', () => {
	it( 'Renders the html',  () =>{
		const content = {
			renderer: '<div>Hi Roy</div>'
		};
		const component = renderer.create(
			<ContentDisplay content={content}/>
		);
		expect(component.toJSON()).toMatchSnapshot();
	});
})
