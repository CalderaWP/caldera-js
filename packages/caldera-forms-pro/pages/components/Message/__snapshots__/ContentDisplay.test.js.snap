// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentDisplay Renders the html 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": undefined,
    }
  }
/>
`;
